### ISSUE TITLE
 get /characters/{character_id}/mail/ 

### ISSUE THREAD

**Issue by swerrie** [ccpgames/esi-issues#237]
Is there a way to tell if the mail is trashed?

**Comment by swerrie** [comment]
adding trash  to: 
get /characters/{character_id}/mail/labels/ 
will fix this issue.

Anyone else opinions?

**Comment by polarina** [comment]
If I recall correctly, whether a mail is trashed or not is not indicated by a label.

**Comment by ddouglas493** [comment]
@polarina This is correct, but how can we move something to Trash via the API if there is no label to assign to it? Is that an acceptable risk that user take by using an OGB Mail Client

**Comment by ddouglas493** [comment]
Is there any update to this? Now that EVEMail has gone into production, I have no way of determining if a mail in currently in the trash or not.

**Comment by cvweiss** [comment]
An evemail with no labels will show up in the Trash "folder" in game.
